Functions/Administration/xASL_adm_SortDicomToFolders.m

```diff
@@ -0,0 +1,117 @@
+function xASL_adm_SortDicomToFolders(pathDICOM, nDirLayers, bUseDCMTK, bVerbose)
+%xASL_adm_SortDicomToFolders Sorts DICOMs in the pathDICOM folder to directories based on the sequence name
```

Comment:
Do you mean based on SeriesDescription (or ProtocolName if SeriesDescription doesn't exist)?

Reply:
OK, I have extended the explanation here: you can also move this to the longer explanation/definition.
And you can resolve :)

%xASL_adm_SortDicomToFolders Sorts DICOMs in the pathDICOM folder to directories based on the sequence name as defined in DICOM tags "SeriesDescription" or "ProtocolName". Note that these are most likely the names set in the protocol on the scanner, or changed by the radiographics.

Reply:
Done